source/MaterialXCross/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+file(GLOB_RECURSE materialx_source "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp")
+file(GLOB_RECURSE materialx_header "${CMAKE_CURRENT_SOURCE_DIR}/*.h")
+
+function(assign_source_group prefix)
+    foreach(_source IN ITEMS ${ARGN})
+        if (IS_ABSOLUTE "${_source}")
+            file(RELATIVE_PATH _source_rel "${CMAKE_CURRENT_SOURCE_DIR}" "${_source}")
+        else()
+            set(_source_rel "${_source}")
+        endif()
+        get_filename_component(_source_path "${_source_rel}" PATH)
+        string(REPLACE "/" "\\" _source_path_msvc "${_source_path}")
+        source_group("${prefix}\\${_source_path_msvc}" FILES "${_source}")
+    endforeach()
+endfunction(assign_source_group)
+
+assign_source_group("Header Files" ${materialx_header})
+assign_source_group("Source Files" ${materialx_source})
+
+set(MATERIALX_GLSLANG_DIR "" CACHE STRING "Location of glslang.")
+set(MATERIALX_SPIRV_CROSS_DIR "" CACHE STRING "Location of SPIRV-Cross.")
+
+# Please note that glslang modifies the default install directory so it's best
+# to set it explicitly with CMAKE_INSTALL_PREFIX in the build script.
+if (IS_DIRECTORY ${MATERIALX_GLSLANG_DIR})
+    add_subdirectory(${MATERIALX_GLSLANG_DIR} glslang)
```

Comment:
Just added as a subdirectory which ensures compilation settings match. Building `glslang` separately may be more convenient because of the CMake side effects described in the comments.